components/klaudsolcms/AppContentBuilderTable.js

```diff
@@ -59,8 +59,13 @@ const AppContentBuilderTable = ({typeSlug}) => {
 
     const onUpdateField = evt => {
      if(formRef.current) formRef.current.handleSubmit();
+      setLoading(true)
```

Comment:
Conventionally, we put `setLoading(true)` inside the `try` block before the API call / `slsFetch`. This is they same block with a `setLoading(false)` on the `finally` block. We do this so that it's easier to visually follow when the loader shows or hides itself.

Can we try moving this on line 81?